client/src/store/index.js

```diff
@@ -33,10 +34,49 @@ export const actions = {
       dispatch('getDailyData');
     });
   },
+  getSelectedLanguagesWithFlags({ commit, dispatch }) {
+    commit('setLoading', true);
+    axios
```

Comment:
I think wee can use the `selectedLanguages` state for this instead of calling the backend everytime? By the time this is called in `retrieveDailyData`, `selectedLanguages` should already be filled.